src/blog/2023/03/3-quick-node-red-tips-4.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 3 Quick Tips for Using Node-RED, Part 4
+subtitle: Save yourself time when working on Node-RED with these three tips.
+description: Save yourself time when working on Node-RED with these three tips.
+date: 2023-03-13 12:00:00
+authors: ["rob-marcer"]
+image: /images/blog/nr-quicktips.jpg
+---
+
+There is usually more than one way to complete a given task in software, and Node-RED is no exception. In each of this series of blog posts, we are going to share three useful tips to save yourself time when working on your flows.
+<!--more-->
+
+
+### 1. Use the Smooth node to get the minimum and maximum values of your payloads
+
+When taking data in from sensors sometimes a spurious value can be sent into your flow. This can result in oddities in a graph or even misfiring of actions such as turning on a heating system. the Smooth custom node allows you to store the min and max of a payload for the last few messages received.
```

Comment:
```suggestion
When taking data in from sensors sometimes a spurious value can be sent into your flow. This can result in oddities in a graph or even misfiring of actions such as turning on a heating system. The Smooth custom node allows you to store the min and max of a payload for the last few messages received.
```